Python/Product/PythonTools/PythonTools/Project/PythonProjectFactory.cs

```diff
@@ -193,12 +203,30 @@ public PythonProjectFactory(IServiceProvider/*!*/ package)
 
             // Importing a targets file from 2.1 Beta
             ProcessImportsFrom21b(projectXml, log);
+
+            // Add missing WebBrowserUrl property
+            ProcessMissingWebBrowserUrl(projectXml, log);
         }
 
         private static bool IsPtvsTargetsFileProperty(ProjectPropertyElement p) {
             return p.Name == "PtvsTargetsFile";
         }
 
+        private static void ProcessMissingWebBrowserUrl(ProjectRootElement projectXml, Action<__VSUL_ERRORLEVEL, string> log) {
+            var launcher = projectXml.Properties.LastOrDefault(p => p.Name == "LaunchProvider");
+            if (launcher == null ||
+                launcher.Value != "Web launcher" ||
```

Comment:
I should have put the comment on the line below, by 2 I'm talking about "Web launcher" and "Django launcher".  `Iauncher.Value` can't be both so one of them will invariably evaluate to true.